test/spec/openapi/option.js

```diff
@@ -607,4 +607,39 @@ test('uses examples if has multiple array examples', t => {
   })
 })
 
+test('uses examples if has property required in body', t => {
+  t.plan(4)
+  const fastify = Fastify()
+
+  fastify.register(fastifySwagger, openapiOption)
+
+  const opts = {
+    schema: {
+      body: {
```

Comment:
Could you move the `body` object to a `const` and check that the `required` field is not changed after the ready event?

```
const body = { required: ['asd'] }

...
   schema: { body }

t.same(body.required, ['asd'], 'must not become a boolean')
```

Reply:
Thank you, I updated spec with this check.